<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>918</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1463</width>
    <height>918</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1463</width>
    <height>918</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Binary Search Tree</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: aliceblue;
}

QPushButton
{
	color: brown;
	font: 15pt 'Consolas';
	background-color: aqua;
	border: 5px solid black;
	border-radius: 40px;
}

QPushButton:hover
{
	color: darkred;
	background-color: aquamarine;
	border: 3px solid black;
}

QPushButton:pressed
{
	color: firebrick;
	background-color: cornflowerblue;
	border: 7px solid black;
}

QGraphicsView
{
	background-color: wheat;
}

QTextEdit
{
	color: black;
	font: 20pt 'Consolas';
	border: 5px solid black;
	border-radius: 10px;

}

QLabel
{
	color: black;
	font: 20pt 'Monotype Corsiva';
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNodeBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>110</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить узел</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Бинарное дерево поиска</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="forwardPassage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Прямой обход дерева</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteNodeBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>210</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить узел</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteTreeBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить дерево</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findNodeBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>410</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск элемента</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchModeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Вертикальная печать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="forwardText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="symmetricPassage">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>700</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Обратный обход дерева</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="symmetricText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>780</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="reverseText">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>780</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="reversePassage">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>700</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Симметричный обход дерева</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="balanceTreeBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>510</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбалансировать дерево</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>1111</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>2000.000000000000000</width>
      <height>2000.000000000000000</height>
     </rectf>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1463</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
